packages/@ember/-internals/glimmer/lib/utils/references.ts

```diff
@@ -207,7 +220,25 @@ export class RootPropertyReference extends PropertyReference
       (this.tag.inner as TwoWayFlushDetectionTag).didCompute(_parentValue);
     }
 
-    return get(_parentValue, _propertyKey);
+    let parent: any;
+    let tracker: any;
+
+    if (EMBER_METAL_TRACKED_PROPERTIES) {
+      parent = getCurrentTracker();
+      tracker = setCurrentTracker();
+    }
+
+    let ret = get(_parentValue, _propertyKey);
+
+    if (EMBER_METAL_TRACKED_PROPERTIES) {
+      setCurrentTracker(parent!);
```

Comment:
I’m surprised that you need the `!` here, if the function allows undefined we should just fix the signature?